cli/src/main/scala-2.12/coursier/cli/options/CommonOptions.scala

```diff
@@ -98,7 +98,13 @@ final case class CommonOptions(
     cacheOptions: CacheOptions = CacheOptions(),
 
   @Help("Retry limit for Checksum error when fetching a file")
-    retryCount: Int = 1
+    retryCount: Int = 1,
+
+  @Help("Specify if artifacts with urls with 'file://' protocol should be cached. " +
+    "Defaults to false, meaning artifacts with urls beginning with 'file://' will " +
+    "not be cached unless this is set to True. In this case, please use the 'local://' protocol.")
+  @Short("cla")
+    cacheFileArtifacts: Boolean = false
```

Comment:
By `cla`, do you mean `cacheLocalArtifacts`?

Reply:
Maybe `cfa` is better here? I just wanted something shorter than `cacheFileArtifacts` for when I was testing it on the command line. I'm open to suggestions!

Reply:
Either works as long as it's consistent :)